terraform/modules/quorum-cluster-region/quorum-cluster-region-backup-event.tf

```diff
@@ -0,0 +1,399 @@
+
+# This is to be placed in quorum-cluster-region
+
+provider "archive" {}
+provider "random" {}
+provider "null" {}
+
+data "local_file" "private_key" {
```

Comment:
This will break TFE because the file won't exist. You need to add the option to provide the key directly in the variables file. If you keep the option to load it from a file there will need to be a `count` variable that ensures this data source is not created if the key is passed directly. I've dealt with this before so you should be able to look to my handling of SSH keys for examples.

Reply:
Fixed